docs/configure-and-manage/manage/qlight-node.md

```diff
@@ -73,10 +74,9 @@ The valid values are:
 You can add an encryption layer on the qlight client-server communication.
```

Comment:
Change this heading to "Configure TLS" or similar.

Reply:
Fixed